### ISSUE TITLE
Pictures inserted in nbs are not included in the repo

### ISSUE THREAD

**Issue by qibin-google** [colaberry/odsceast2018#1]
This makes the notebooks very hard to follow

**Comment by vsathwikm** [comment]
Thanks for bringing this to our notice. I will fix this a.s.a.p

**Comment by vsathwikm** [comment]
@qibin-google I have added all images with appropriate paths. Github does not always consistently render Jupyter notebooks, so you may not always be able to see the images in the notebook. We recommend accessing this course on https://refactored.ai/ (First notebook which starts the ODSCeast 2018 track: https://refactored.ai/track/bayesian-networks-using-pgmpy/courses/Bayesian_Theory.ipynb).

Refactored is an open platform and can be accessed by a simple social media or email sign in.
